app/models/solr_document.rb

```diff
@@ -149,4 +149,64 @@ def degree_name
   def graduation_year
     self[Solrizer.solr_name('graduation_year')]
   end
+
```

Comment:
something worth considering would be to take an array of the field names and dynamically define these standard methods for them all at once;

```
%w(time_required typical_age_range learning_resource_type).each do |property_name|
  define_method property_name do
    self[Solrizer.solr_name(property_name)]
  end
end
```